public/app/mailers/request_mailer.rb

```diff
@@ -4,42 +4,39 @@ def request_received_email(request)
 
     @request = request
 
-    mail(from: from_address(request.repo_code), 
+    mail(from: email_address(@request),
          to: user_email,
          subject: I18n.t('request.email.subject', :title => request.title))
   end
 
   def request_received_staff_email(request)
     @request = request
 
-    mail(from: from_address(request.repo_code),
-         to: to_address(request.repo_code),
+    mail(from: email_address(@request),
+         to: to_address(@request, :to),
          subject: I18n.t('request.email.subject', :title => request.title))
   end
 
-  def email_pdf_finding_aid(recipient_email, repo_code, record_title, suggested_filename, pdf_path)
-    attachments[suggested_filename] = File.read(pdf_path)
+  # TODO: not implemented
+  # def email_pdf_finding_aid(request, recipient_email, suggested_filename, pdf_path)
```

Comment:
@mark-cooper I think this needs to be implemented prior to accepting this pull request.

Reply:
@lmcglohon it already wasn't implemented (for whatever reason). I'm just making that clear.